helm-charts/chatqna/vllm-openvino-values.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+vllm:
+  openvino_enabled: true
```

Comment:
Does not confirm to Helm best practices: https://helm.sh/docs/chart_best_practices/values/

Should be either `openvinoEnabled: true`, or `openvino: true`.